content/en/about.md

```diff
@@ -67,8 +68,7 @@ NumPy receives direct funding from the following sources:
 
 Institutional Partners are organizations that support the project by employing people that contribute to NumPy as part of their job. Current Institutional Partners include:
 
-- UC Berkeley (Stéfan van der Walt, Ross Barnowski)
```

Comment:
I was going to approve this, but then remembered that I am still hired by Berkeley :)